src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
+If you sign up using your email address, your next step will be to confirm the
+email address by entering a code sent to you via email. You can paste the code
+into any of the boxes rather than typing each number individually.
 
 ![Confirm Email](./images/EmailConfirmation1.png)
 
-## CodeStream organizations [#organizations]
-
-A CodeStream organization is where you and your teammates will discuss code. Similar to a Slack workspace, all of the developers in your company are in the same CodeStream organization. CodeStream's activity feed keeps things relevant for you by showing activity related to the code you have open in your IDE. 
-
-The discussions about code build up a knowledge base that is a company-wide resource, so the only reason to have multiple organizations on CodeStream is if you truly need separation. For example, you might have an organization for your day job and another for an open-source project you work on. Or maybe you're a consultant that's a member of different CodeStream organizations for each of your clients.
-
 ## Create or join an organization [#create-org]
 
-If you're invited to join an organization on CodeStream, sign up with the same email address the invitation was sent to. You'll automatically be added to that organization.
+A CodeStream organization is where you and your teammates will discuss code.
+Similar to a Slack workspace, all of the developers in your company can be in
+the same CodeStream organization. 
 
-If you're the first person from your company to sign up for CodeStream you can go ahead and create a new organization. Otherwise, there may be existing CodeStream organizations available for you to join based on your email domain. If you think your company is already on CodeStream, but don't see an organization to join, make sure that you've signed up with your work email.
+If you're invited to join an organization on CodeStream, sign up with the same
+email address the invitation was sent to and you'll automatically be added to
+that organization. Alternatively, there may be existing CodeStream organizations
+available for you to join based on your email domain.
 
 ![Create or join an organization](./images/CreateOrJoinOrg.png)
 
-If you decide to create an organization you'll be asked to give it a name and, if you signed up with your work email address, anyone else with that email domain can join the organization.
-
-![Create an organization](./images/CreateOrg.png)
+If you weren't invited, and there are no existing organizations for you to join,
+a new organization will be created for you.
 
-## Invite your team [#invite]
+## Invite your teammates [#invite]
 
-Collaboration is a team sport so invite your teammates to join you on CodeStream.
+Collaboration is a team sport so invite your teammates to join your CodeStream
+organization.
```

Comment:
I think it'd be great to get more specific about what value you get out of inviting your teammates to CodeStream. Personally, I don't find "team sport" very compelling :D


Off the top of my head, adding teammates gets you increased collaboration through:
* code comments
* feature requests
* pull requests
* other things I'm not considering?

Reply:
Will address in next commit, although this isn't really the article to be selling that. Especially since we've already mentioned above that a CS org is for collaborating with teammates.